Modules/Translator.cs

```diff
@@ -179,6 +179,7 @@ public static string GetString(string s, Dictionary<string, string> replacementD
         var langId = TranslationController.InstanceExists ? TranslationController.Instance.currentLanguage.languageID : SupportedLangs.English;
         if (console) langId = SupportedLangs.English;
         if (Main.ForceOwnLanguage.Value) langId = GetUserTrueLang();
+        s = Utils.GetAddOnLabel(s); // Add-On label system for add-ons that can be configured both positive & negative
```

Comment:
s = "SlowInfo"; 

test showed positive, so Ig this has to run once the game loaded, so I will have to figure that out.